src/huntsman/pocs/camera/pyro/client.py

```diff
@@ -46,6 +41,9 @@ def __init__(self, uri, name='Pyro Camera', model='pyro', port=None, *args, **kw
         self.connect()
 
     # Properties
+    @property
+    def _proxy(self):
+        return Proxy(self._uri)
```

Comment:
Since this is in fact a pyro class, I could make an argument for this being non-private (i.e. `proxy` vs `_proxy`).

Reply:
You have a point. Let's leave this for a later discussion.